Pod/Classes/Optional Data Sources/VOKFetchedResultsDataSource.h

```diff
@@ -224,6 +224,33 @@ NS_ASSUME_NONNULL_BEGIN
                          delegate:(nullable id <VOKFetchedResultsDataSourceDelegate>)delegate;
 
 /**
+ Setup this data source.
+ 
+ @param coreDataManager    VOKCoreDataManager object to use as the core data manager
+ @param predicate          Predicate used to filter the objects displayed
+ @param cacheName          Name for the fetched results controller cache
+ @param tableView          Table view to display in. The data source and delegate are set to the newly-created data source.
+ @param sectionNameKeyPath Keypath to use to group results in sections
+ @param sortDescriptors    Sort descriptors to sort the objects
+ @param managedObjectClass NSManagedObject subclass to fetch
+ @param batchSize          Batch size to use for the fetch request fetchBatchSize.
+ @param fetchLimit         Fetch limit to use for the fetch request fetchLimit.
+ @param delegate           Delegate to notify of object selection/deselection and whether the fetch has results
+ 
+ @return New data source
+ */
+- (instancetype)initWithCoreDataManager:(VOKCoreDataManager *)coreDataManager
+                              predicate:(NSPredicate *)predicate
+                              cacheName:(NSString *)cacheName
+                              tableView:(UITableView *)tableView
+                     sectionNameKeyPath:(NSString *)sectionNameKeyPath
+                        sortDescriptors:(NSArray *)sortDescriptors
```

Comment:
Whoops.  I really wish Xcode's autocomplete on the `.m` side would carry over the nullability annotations, etc., from the `.h` side.  (This was copied back from the `.m`, hence missing all that stuff.)